core/server/users.js

```diff
@@ -157,6 +157,9 @@ Meteor.methods({
 
     analytics.identify(Meteor.user());
   },
+  checkOnboarded() {
```

Comment:
"check" doesn't seem appropriate here, we're not "checking" something (as in "verifying")